
<launch>
	
    <arg name="host"/>
    <arg name="port"/>
    <arg name="drop_rate"/>
    <arg name="arm"/>
    <arg name="ros_rate"/>
    <arg name="namespace"/>



   <group ns="$(arg arm)">
  	
      <node name="rosbridge_upstream" pkg="irob_robot"
      type="dvrk_rosbridge_upstream.py" output="screen">
  		
          <param name="host" type="string" value="$(arg host)" />
          <param name="port" type="int" value="$(arg port)" />
          <param name="drop_rate" type="double" value="$(arg drop_rate)" />
          <param name="ros_rate" type="double" value="$(arg ros_rate)" />
          <param name="arm" type="string" value="$(arg arm)" />
          <param name="namespace" type="string" value="$(arg namespace)" />
				
			</node>
    </group>


</launch>
